sandbox/Test.scala

```diff
@@ -1,13 +1,15 @@
-import scalanative.native._
+import scalanative.native._, FunctionPtr._
```

Comment:
@sjrd Any idea why implicits in FunctionPtr are not resolved automatically? How does this work in SJS for ThisFunctions?